app/src/main/java/com/github/h3lp3rs/h3lp/notification/NotificationService.kt

```diff
@@ -0,0 +1,123 @@
+package com.github.h3lp3rs.h3lp.notification
+
+import android.annotation.SuppressLint
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.graphics.BitmapFactory
+import android.os.Build
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.github.h3lp3rs.h3lp.R
+
+/**
+ *  Static class containing useful function to throw notification.
+ *  The notification are stylized with H3LP app style
+ *  @warning allways make sure to use {@link createNotificationChannel} method to make sure the notification channel is initialized
+ */
+class NotificationService {
+
+    companion object {
+        private const val CHANNEL_ID = "Help_channel_id"
+        private var notificationId= 0
+
+
+
+        /**
+         * Create the notification channel for HELP application
+         * Should be called before sending any notification
```

Comment:
Since it should always be called in order to initialize the channel, it would make sense to find a way to do that automatically, either by putting it in the companion object so that it's called once when the app is launched, or in the constructor so that we call it every time we create an instance of NotificationService

Reply:
That's true.
I was thinking that this will be called when constructing the part that will be running in backgroung of our app reponsible for emergency notification

Reply:
After some reflexion here is what i come up with:
- putting it in init block is impossible since it need the context of an app to be started.
- I read this is always better to recreate the chanel before sending a notification. Because sometime android remove it. That not bad for permormance because createNotificationChannel always check if the notififcation channel is already created and doen't recreated it then
- If we still want to initialize it once I think we could call this at the same time than the database listener which will be normally be done next week